commit 287fd7890e08f8dbcacc8e471812d3f3965997e8 (refs/changes/28/3528/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2018-03-02T01:17:31+00:00 (1 year, 7 months ago)
    
    TRITON-200 sdc-docker cli-nfs-shared-volumes-disabled.test.js is flaky

diff --git a/test/integration/cli-nfs-shared-volumes-disabled.test.js b/test/integration/cli-nfs-shared-volumes-disabled.test.js
index 71df3b6..b832585 100644
--- a/test/integration/cli-nfs-shared-volumes-disabled.test.js
+++ b/test/integration/cli-nfs-shared-volumes-disabled.test.js
@@ -34,7 +34,7 @@ var NFS_SHARED_VOLUME_NAMES_PREFIX =
     testVolumes.getNfsSharedVolumesNamePrefix();
 var SAPI_CLIENT;
 var SAPI_APP;
-var STATE_RETRIES = 120;
+var STATE_RETRIES = 180;
 
 // wait for /admin/config to have state either 'enabled' or 'disabled' for
 // experimental_docker_nfs_shared_volumes
@@ -137,8 +137,8 @@ test('setup', function (tt) {
                         + app.metadata.experimental_docker_nfs_shared_volumes);
                 }
 
-                if (app.metadata.experimental_docker_nfs_shared_volumes ===
-                    false) {
+                if (app.metadata.experimental_docker_nfs_shared_volumes !==
+                    true) {
                     t.comment('NFS volumes support already disabled, no need '
                         + 'to turn it off');
                     t.end();
